environment/src/main/java/com/sequenceiq/environment/experience/liftie/ExperiencesByLiftie.java

```diff
@@ -0,0 +1,115 @@
+package com.sequenceiq.environment.experience.liftie;
+
+import static com.sequenceiq.cloudbreak.util.ConditionBasedEvaluatorUtil.throwIfTrue;
+import static com.sequenceiq.environment.experience.liftie.LiftieIgnorableClusterStatuses.DELETED;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Set;
+
+import javax.validation.constraints.NotNull;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.environment.environment.dto.EnvironmentExperienceDto;
+import com.sequenceiq.environment.experience.Experience;
+import com.sequenceiq.environment.experience.ExperienceSource;
+import com.sequenceiq.environment.experience.api.LiftieApi;
+import com.sequenceiq.environment.experience.config.LiftieWorkloadsConfig;
+import com.sequenceiq.environment.experience.liftie.responses.ClusterView;
+import com.sequenceiq.environment.experience.liftie.responses.ListClustersResponse;
+
+@Component
+public class ExperiencesByLiftie implements Experience {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ExperiencesByLiftie.class);
+
+    private final ListClustersResponseValidator listClustersResponseValidator;
+
+    private final LiftieApi liftieApi;
+
+    private final Set<LiftieWorkload> workloads;
+
+    public ExperiencesByLiftie(LiftieWorkloadsConfig workloadConfig, LiftieApi liftieApi,
+            ListClustersResponseValidator listClustersResponseValidator) {
+        this.listClustersResponseValidator = listClustersResponseValidator;
+        this.liftieApi = liftieApi;
+        this.workloads = identifyConfiguredWorkloads(workloadConfig);
+    }
+
+    @Override
+    public int getConnectedClusterCountForEnvironment(EnvironmentExperienceDto environment) {
+        throwIfTrue(environment == null, () -> new IllegalArgumentException(EnvironmentExperienceDto.class.getSimpleName() + " cannot be null!"));
+        List<ClusterView> clusterViews = getClusterViewsForWorkloads(environment.getName(), environment.getAccountId());
```

Comment:
Can more logs could be added to the `getClusterViewsForWorkloads` method and or it's subcalls?
The other public method `delete...` is covered nicely with logs.

Reply:
ok